AdyenComponents/Atome/AtomeComponent.swift

```diff
@@ -0,0 +1,93 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import UIKit
+
+/// A component that provides a form for Atome payment.
+public final class AtomeComponent: AbstractPersonalInformationComponent {
+
+    /// Configuration for Atome Component
+    public typealias Configuration = PersonalInformationConfiguration
+
+    private enum ViewIdentifier {
+        static let billingAddress = "billingAddressItem"
+    }
+
+    // MARK: - Items
+
+    /// :nodoc:
+    private let personalDetailsHeaderItem: FormLabelItem
+
+    // MARK: - Initializers
+
+    /// Initializes the Atome component.
+    /// - Parameters:
+    ///   - paymentMethod: The Atome payment method.
+    ///   - apiContext: The component's API context.
+    ///   - configuration: The component's configuration.
+    public init(paymentMethod: PaymentMethod,
+                apiContext: APIContext,
+                configuration: Configuration = .init()) {
+        personalDetailsHeaderItem = FormLabelItem(text: "", style: configuration.style.sectionHeader)
+
+        let fields: [PersonalInformation] = [
+            .custom(CustomFormItemInjector(item: FormSpacerItem(numberOfSpaces: 2))),
+            .custom(CustomFormItemInjector(item: personalDetailsHeaderItem.addingDefaultMargins())),
+            .firstName,
+            .lastName,
+            .phone,
+            .address,
+            .custom(CustomFormItemInjector(item: FormSpacerItem(numberOfSpaces: 1)))
+        ]
+
+        super.init(paymentMethod: paymentMethod,
+                   apiContext: apiContext,
+                   fields: fields,
+                   configuration: configuration)
+
+        setupItems()
+    }
+
+    // MARK: - Private
+
+    /// :nodoc:
+    private func setupItems() {
+        personalDetailsHeaderItem.text = localizedString(.boletoPersonalDetails, configuration.localizationParameters)
+        phoneItem?.title =  localizedString(.phoneNumberTitle, configuration.localizationParameters)
+    }
+
+    // MARK: - Public
+
+    /// :nodoc:
+    override public func submitButtonTitle() -> String {
+        localizedString(.continueTitle, configuration.localizationParameters)
+    }
+
+    /// :nodoc:
+    override public func createPaymentDetails() -> PaymentMethodDetails {
+        guard let firstName = firstNameItem?.value,
+              let lastName = lastNameItem?.value,
+              let telephoneNumber = phoneItem?.value,
+              let billingAddress = addressItem?.value else {
+            fatalError("There seems to be an error in the BasicPersonalInfoFormComponent configuration.")
```

Comment:
maybe it's better to have this as an `assertionFailure`, rather than `fatalError`

Reply:
@kaphacius assertionFailure() do not guarantee that execution will stop if the assert is triggered and I guess we would like to stop the execution as all this information is required for the payment. Please let me know if I am wrong here.

Reply:
we should't crash the merchant app, so I am afraid Yurii has a point, `assertionFailure` makes more sense IMO

Reply:
Okay then I will change this to assertionFailure. Thanks for pointing this out.

Reply:
we have `Adyen. assertionFailure()` for this.
But I would make this check on `didSubmit` so we can return it to shopper as validation error

Reply:
I have modified the code as per @descorp suggestion.